google/cloud/pubsub_v1/gapic/transports/publisher_grpc_transport.py

```diff
@@ -116,8 +116,12 @@ def channel(self):
     def create_topic(self):
         """Return the gRPC stub for :meth:`PublisherClient.create_topic`.
 
-        Creates the given topic with the given name. See the resource name
-        rules.
+        Deletes the topic with the given name. Returns ``NOT_FOUND`` if the
```

Comment:
@vam-google It looks like there is something strange happening with protoc-docs-plugin. These docstrings are getting attached to the wrong function (I've seen it in a few other libraries as well). I'm going to try to reproduce this on my own machine later today.

Reply:
@busunkim96, did you figure out what was happening here?